### ISSUE TITLE
When "server_connections > server_failure_limit"

### ISSUE THREAD

**Issue by deep011** [twitter/twemproxy#430]
We set server_connections bigger than server_failure_limit.
Sometime the connections to mc/redis are not stable, suddenly mc/redis close some connections that  more then server_failure_limit but less than server_connections, this mc/redis will be eject!
@manjuraj, maybe something is wrong?

**Comment by manjuraj** [comment]
server_connections has nothing to do with server_failure_limit; those two parameters are unrelated

**Comment by deep011** [comment]
For example, server_connections is 20 and server_failure_limit is 3, then the mc
close 3 connections with twemproxy for some reasons, at this time twemproxy will eject this mc, but actually this mc is not go down.

**Comment by manjuraj** [comment]
Good point - this is bug or a new feature request depending on the way you look at it; failure is accrrued on the server level and not on a connection level

**Comment by digitalprecision** [comment]
@deep011 How are you guys handling the situation? We are running into something similar where hosts are being auto-ejected for no apparent reason, and it may be due to this issue.

**Comment by deep011** [comment]
Set "server_connections <= server_failure_limit".
